pkg/crc/cluster/cluster.go

```diff
@@ -92,6 +93,33 @@ func GetRootPartitionUsage(sshRunner *ssh.Runner) (int64, int64, error) {
 	return diskSize, diskUsage, nil
 }
 
+func UpdateSSHKeyToMachineConfig(ocConfig oc.Config, sshPublicKeyPath string) error {
+	sshPublicKeyByte, err := ioutil.ReadFile(sshPublicKeyPath)
+	if err != nil {
+		return err
+	}
+	sshPublicKey := strings.TrimRight(string(sshPublicKeyByte), "\r\n")
+	if err := WaitForOpenshiftResource(ocConfig, "machineconfigs"); err != nil {
+		return err
+	}
+	stdout, _, err := ocConfig.RunOcCommand("get", "machineconfigs", "99-master-ssh", "-o", `jsonpath='{.spec.config.passwd.users[0].sshAuthorizedKeys[0]}'`)
+	if err != nil {
+		return err
+	}
+	if stdout == string(sshPublicKey) {
+		return nil
+	}
+	logging.Info("Updating SSH key to machine config resource...")
```

Comment:
`Adding public SSH key to the cluster` or something similar is easier to read/understand imo.